examples/threelang/tfgrid/vm.md

```diff
@@ -0,0 +1,33 @@
+# Machines Example
+
+- This example deployes, gets, udpates, and deletes a network of machines on the tfgrid.
+
+!!tfgrid.core.login
+	mnemonic: 'YOUR MNEMONICS'
+	network: dev
+
+!!tfgrid.sshkeys.new
+	name: default
+	ssh_key: 'YOUR SSH KEY'
+
+!!tfgrid.machines.create
```

Comment:
Rename to tfgrid.machine.create instead. Do that for all of other actions related to deploying a vm too.